### ISSUE TITLE
Admin Event Hook for Flex Object

### ISSUE THREAD

**Issue by ChristianaHoJo** [trilbymedia/grav-plugin-flex-objects#148]
Hello!

Can you give me the correct Admin Event Hook that I can use to intervene when saving Flex Objects to extend the functionality in my plugin? The “onAdminSave” event does not work for me at this point.
I am looking forward to help!

Greetings, Christiana

**Comment by mahagr** [comment]
Yes. That said, it's also possible to have an event call every time the object is updated, regardless of where it happens.
